CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.37.19] - 2022-08-31
```

Comment:
We probably need to bump minor version for this. 29.38.0. Because it introduces some backward incompatible changes, and we would like this to get user attention at gradle build time.

Reply:
Done